### DISCUSSION TITLE
Sub stream with Wyze V4

### DISCUSSION THREAD

**Discussion by rspierenburg** [mrlt8/docker-wyze-bridge#1271] [Q&A]
I'm currently running Docker Wyze Brige along with Frigate on my Home Assistant server.  I have two V4s and a V3.  Frigate is using an abhorrent amount of CPU resources and so it was recommended to use a sub stream with lower resolution to do the detection.  So I checked off the box to "Enable Sub-Stream" but it seems to only work with the V3 camera.  Are the V4s not capable of running a substream?

Rob

**Comment by mrlt8** [comment]
Sub stream support depends on the firmware. Can you try forcing a sub stream to see if it works with the firmware on your v4?

Camera Specific Options:

```yaml
- CAM_NAME: My Camera
  SUBSTREAM: true
```

**Reply by jovanoo** [reply]
Hi did this work? I struggle understanding how I can get a livestream going in 360p on the V4.
